default/en/cat_printed.md

```diff
@@ -66,3 +79,68 @@ PLATTNER à ROTTERDAM." **New record required** (different publication status).
    field "Material held" please note which symphony (livre) you have.
    2. Option 2, a better solution, but takes more time: **Notify the Editorial Center** about this record and we will split
    the record into 3, one record for each symphony. Then, add your holdings to the record for the symphony you have.
+
+## Standardized titles for printed music
+
+   **Distinctive titles**
```

Comment:
These should probably be level-3 headings:

```
### Distinctive titles
```

And not indented.

Reply:
Yes, thanks - in this PR, I only joined files. I haven't edited them yet.